blockchain/genesis/genesis.go

```diff
@@ -61,6 +61,7 @@ func defaultConfig() Genesis {
 			InitBalanceMap: make(map[string]string),
 		},
 		Poll: Poll{
+			PollMode:                         "governanceMix",
```

Comment:
certain tests used lifelong
i think you'll need to set "lifelong" in those tests?